modules/workers/nodepools.tf

```diff
@@ -57,7 +57,7 @@ resource "oci_containerengine_node_pool" "workers" {
       for_each = var.cni_type == "npn" ? [1] : []
       content { # VCN-Native requires max pods/node, nsg ids, subnet ids
         cni_type          = "OCI_VCN_IP_NATIVE"
-        max_pods_per_node = min(max(var.max_pods_per_node, 1), 110)
+        max_pods_per_node = each.value.max_pods_per_node
```

Comment:
If the node pool does not specify this value, it should take the global default.

Reply:
Yes, the global value is defined in the `worker_pool_defaults`, [here](https://github.com/oracle-terraform-modules/terraform-oci-oke/pull/904/files#diff-54c5df4ead89ff4bd0fb96b9fba84ff8206c61a5b27be4c7f8f2d6124e810a3fR42).

Reply:
I'm not seeing where you are taking the global default if the value in the nodepool is not specified. 

You network options look like this:

```
        max_pods_per_node = each.value.max_pods_per_node
```

You should have a coalesce here: 

```
        max_pods_per_node = coalesce(each.value.max_pods_per_node, local.max_pods_per_node)
```